saleor/graphql/account/mutations/account.py

```diff
@@ -162,12 +164,26 @@ def save(cls, info: ResolveInfo, user, cleaned_input):
             if site.settings.enable_account_confirmation_by_email:
                 user.is_active = False
                 user.save()
+
+                # Notifications will be deprecated in the future
                 notifications.send_account_confirmation(
                     user,
                     cleaned_input["redirect_url"],
                     manager,
                     channel_slug=cleaned_input["channel"],
                 )
+
+                token = default_token_generator.make_token(user)
+                params = urlencode({"email": user.email, "token": token})
+                confirm_url = prepare_url(params, cleaned_input["redirect_url"])
+
+                cls.call_event(
+                    manager.account_confirmation_requested,
+                    user,
+                    cleaned_input["channel"],
+                    token,
+                    confirm_url,
+                )
```

Comment:
It's still only sent if account confirmation is required.

Reply:
Done, additionally, as @fowczarek mentioned in #13170 I make notify and webhook event to use the same token.